### PR TITLE
Add Participant Status documentation

### PR DESCRIPTION
**Pull request by mholzer17**
Description
----
Add documentation for appointment participant status read and patch updates.

Before Changes:
<details open>
<summary>Appointment:</summary>
<br>
Screen Shot 2022-04-14 at 1 57 38 PM
Screen Shot 2022-04-14 at 1 58 40 PM
Screen Shot 2022-04-14 at 1 58 52 PM
Screen Shot 2022-04-14 at 1 59 06 PM
</details>

After Changes:
<details open>
<summary>Appointment:</summary>
<br>
Screen Shot 2022-04-14 at 1 55 26 PM
Screen Shot 2022-04-14 at 1 56 13 PM
Screen Shot 2022-04-14 at 1 56 35 PM
Screen Shot 2022-04-14 at 1 56 43 PM
Screen Shot 2022-04-14 at 1 56 53 PM
</details>

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/base/workflow/appointment.md [hunk 1]
```diff
@@ -386,7 +415,7 @@ _Implementation Notes_
   * `Appointment.slot` must be a list containing exactly one reference to the Slot in which this appointment is being booked. `Appointment.slot[0].reference` specifies an availability in the Scheduling system, which indicates details such as practitioner, location, and time.
   * `Appointment.participant` must be a list containing exactly one Patient participant.
   * `Appointment.participant.actor` must be a reference to a Patient.
-  * `Appointment.participant.status` must be set to `accepted`.
+  * `Appointment.participant.status` must be set to either `tentative` or `accepted`.
```
Comment (CBrashear) [inline content/millennium/r4/base/workflow/appointment.md:418]: Was this suppose to be in the create section or should it be in read?

Comment (mholzer17) [reply content/millennium/r4/base/workflow/appointment.md]: Reverted here: 3903d357d9d5e189d32b79475ca9fe7442620d1c

**Review by CBrashear — APPROVED** [review]
+1, once update your before photos.

**Review by MadhurRaj — APPROVED** [review]
approving, but make sure to switch the url to https

File: lib/resources/r4/appointment_patch.yaml [hunk 2]
```diff
@@ -233,3 +233,20 @@ operations:
         <li>The value for this patch operation will be applied to both the provider Video Visit link and the patient Video Visit link</li>
         <li>This patch operation must be accompanied by all other Video Visit link patch operations with an op of replace</li>
       </ul>
+  - name: replace-participant-status
+    path: /patient-status
+    operation: replace
+    type: code
+    url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.participant.status
```
[the remark this answers is not in the document]
Comment (mholzer17) [reply lib/resources/r4/appointment_patch.yaml]: Updated here: 5fc92f9ae720412d7611f4d4493a8675b9ce0b20

**Comment by mholzer17** [comment]
After Update:
<details open>
<summary>Appointment:</summary>
<br>
Screen Shot 2022-05-18 at 12 19 30 PM
Screen Shot 2022-05-18 at 12 19 55 PM
Screen Shot 2022-05-18 at 12 20 13 PM
Screen Shot 2022-05-18 at 12 20 29 PM
</details>

**Pull request merged**